pkg/util/util.go

```diff
@@ -196,3 +196,12 @@ func IsRetryable(err error) bool {
 	st, ok := status.FromError(err)
 	return ok && (st.Code() == codes.Unavailable || (st.Code() == codes.Internal && strings.Contains(st.Message(), "RST_STREAM")))
 }
+
+func HandleGrpcError(err error, prefix string) {
```

Comment:
Since this method prints the error, can we rename it to state that funtionanlity